### PR TITLE
Debian9

### PR DESCRIPTION
**Pull request by georgejhunt**
tested on top of startup branch in VMs. Functionally tested -- console, moodle,, kalite,kiwix -- not tested nextcloud

### CODE REVIEW

File: roles/sugarizer/tasks/main.yml [hunk 1]
```diff
@@ -11,14 +11,23 @@
...
+#    - npm
   when: internet_available 
-  tags:
-    - download
+
+- name: Install npm non debian
+  package: name=npm
+           state=present
+  when: internet_available and not is_debian
+
```
Comment (jvonau) [inline roles/sugarizer/tasks/main.yml:23]: when: internet_available and ansible_local.local_facts.os_ver != "debian-9"

File: roles/sugarizer/tasks/main.yml [hunk 2]
```diff
@@ -11,14 +11,23 @@
...
+- name: Install npm on debian -- set up apt sources
+  shell: curl -sL https://deb.nodesource.com/setup_6.x | bash -
+  when: internet_available and is_debian
+
```
Comment (jvonau) [inline roles/sugarizer/tasks/main.yml:27]: when: internet_available and ansible_local.local_facts.os_ver == "debian-9"

File: roles/sugarizer/tasks/main.yml [hunk 3]
```diff
@@ -11,14 +11,23 @@
...
+- name: Actually get it installed
+  command: apt install -y npm
+  when: internet_available and is_debian
```
Comment (jvonau) [inline roles/sugarizer/tasks/main.yml:31]: when: internet_available and ansible_local.local_facts.os_ver == "debian-9"

File: roles/sugarizer/tasks/main.yml (hunk 1 above)
Comment (jvonau) [reply roles/sugarizer/tasks/main.yml:23]: the package npm routine would still needed for debian 8 or is this a running change to fix a sugarizer issue?

**Comment by jvonau** [comment]
Comments in files view, is the change to npm just intended for debian-9?

File: roles/sugarizer/tasks/main.yml (hunk 3 above)
Comment (georgejhunt) [reply roles/sugarizer/tasks/main.yml:31]: Cannot imagine that a newer version of npm will fail on debian8 -- but I don't think I circled back to test -- will do that shortly

**Pull request merged**